### DISCUSSION TITLE
no name info

### DISCUSSION THREAD

**Discussion by ik3ocd** [mbridak/not1mm#128] [General]
good morning from flavio ik3ocd I'm noticing that if I insert a callsign for example mine the name of the station I'm interested in doesn't appear, the configuration on lookup is set correctly on hamqth, am I doing something wrong or is it a problem? thanks for the reply. bye and 73
![no name info ](https://github.com/user-attachments/assets/9daecf42-3c70-477d-bc6f-2012b0055324)
![lookup](https://github.com/user-attachments/assets/29a74229-5b7a-4dca-b523-6d6c649c77f8)

**Comment by mbridak** [comment]
From the photo it looks like the HamQTH lookup is completely broken.
Not because the Name doesn't appear, since there is no code written that does that.
But if you look near the bottom of the screen you'll see the words "Regional Hdg".
This means no gridsquare was retrieved from the attempted lookup.

I should probably rewrite the class anyway. Not sure when/how I broke it.

**Reply by ik3ocd** [reply]
ok thanks for the answer I await your response to this great project. bye and 73 de flavio ik3ocd

**Comment by mbridak** [comment]
@ik3ocd Okay, rewrote part of the hamqth lookup code.
It now works again.
It still doesn't do what you want. but it does do what I want.

I'll still have to noodle on it a bit to maybe do what you want it to do.
Thanks for surfacing the bug.

Cheers, Mike K6GTE

**Comment by mbridak** [comment]
@ik3ocd Okay I've rewrote the look up service. Removing the threading. Added some code to add the looked up name to the name field for the general logging screen.
